lib/index.test.js

```diff
@@ -440,6 +440,7 @@ test.serial('should throw error on double render', t => {
     t.true(renderSpy.calledOnce);
     return t.throwsAsync(
       instance.renderExplicitly(),
+      null,
```

Comment:
Signature of [`thowsAsync`](https://github.com/avajs/ava/blob/master/docs/03-assertions.md#throwsasyncthrower-expectation-message) has changed